public/data/Kunal0007.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Kunal Patil",
+    "bio": "I am Kunal Patil currently pursuing a BTech degree in CS at PCCOE - Pimpri Chinchwad College Of Engineering, Pune. A Frontend Developer | Open-Source Enthusiast",
+    "avatar": "https://github.com/Kunal0007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Kunal0007",
+        "icon": "github"
+      }
+    ]
+}
+  

```

Comment:
```suggestion
```

Reply:
@eddiejaoude told needs a blank line ending at the end of the file

Reply:
The Git diff doesn't display the last line. So you have 2 extra lines here that should be removed. 🙂